pycqed/instrument_drivers/meta_instrument/device_object_CCL.py

```diff
@@ -1,7688 +1,7 @@
-                # acquisition threshold.
-                if calibrate_optimal_weights:
-                    q.calibrate_optimal_weights(analyze=True, verify=False, update=True)
-                if calibrate_optimal_weights and not calibrate_threshold:
-                    log.warning("Updated acq weights but not updating threshold")
-                if calibrate_threshold:
-                    q.measure_ssro(update=True, nr_shots_per_case=2 ** 13)
-
-        self.measure_ssro_multi_qubit(
-            qubits, label=mux_ro_label, result_logging_mode="lin_trans"
-        )
-
-        # if len (qubits)> 2:
-        #     raise NotImplementedError
-
-        # res_dict = mra.two_qubit_ssro_fidelity(
-        #     label='{}_{}'.format(q0.name, q1.name),
-        #     qubit_labels=[q0.name, q1.name])
-        # V_offset_cor = res_dict['V_offset_cor']
-
-        # N.B. no crosstalk parameters are assigned
-        # # weights 0 and 1 are the correct indices because I set the numbering
-        # # at the start of this calibration script.
-        # UHFQC.qas_trans_offset_weightfunction_0(V_offset_cor[0])
-        # UHFQC.qas_trans_offset_weightfunction_1(V_offset_cor[1])
-
-        # # Does not work because axes are not normalized
-        # matrix_normalized = res_dict['mu_matrix_inv']
-        # matrix_rescaled = matrix_normalized/abs(matrix_normalized).max()
-        # UHFQC.upload_transformation_matrix(matrix_rescaled)
-
-        # a = self.check_mux_RO(update=update, update_threshold=update_threshold)
-        return True
-
-    def calibrate_cz_single_q_phase(
-        self,
-        q_osc: str,
-        q_spec: str,
-        amps,
-        q2=None,
-        q3=None,
-        waveform="cz_NE",
-        flux_codeword_park=None,
-        update: bool = True,
-        prepare_for_timedomain: bool = True,
-        MC=None,
-    ):
-        """
-        Calibrate single qubit phase corrections of CZ pulse.
-
-        Parameters
-        ----------
-        q_osc : str
-            Name of the "oscillating" qubit. The phase correction of this
-            qubit will be calibrated.
-        q_spec: str
-            Name of the "spectator" qubit. This qubit is used as the control.
-        amps: array_like
-            Amplitudes of the phase correction to measure.
-        waveform: str
-            Name of the waveform used on the "oscillating" qubit. This waveform
-            will be reuploaded for each datapoint. Common names are "cz_z" and
-            "idle_z"
-
-        Returns
-        -------
-        succes: bool
-            True if calibration succeeded, False if it failed.
-
-        procedure works by performing a conditional oscillation experiment at
-        a phase of 90 degrees. If the phase correction is correct, the "off" and
-        "on" curves (control qubit in 0 and 1) should interesect. The
-        analysis looks for the intersect.
-        """
-
-        if prepare_for_timedomain:
-            self.prepare_for_tim
```

Comment:
I don't understand, if the entire contents of the device object were trivially migrated to the new hardware-abstraction layer (HAL), including all the measurement routines, what is then the point of the HAL? Shouldn't it encompass only abstraction of the interactions with hardware? 

If it encompasses also knowledge of the quantum device and calibration experiments, like is encoded into the qubit object and device object, isn't this just a change in branding? In which case this will further confuse people in the laboratory, who are used to the current structure of code when looking for and building new experiments.